RecastDemo/Source/TestCase.cpp

```diff
@@ -102,9 +102,22 @@ bool TestCase::load(const std::string& filePath)
 	FILE* fp = fopen(filePath.c_str(), "rb");
 	if (!fp)
 		return false;
-	fseek(fp, 0, SEEK_END);
+	if (fseek(fp, 0, SEEK_END) != 0)
+	{
+		fclose(fp);
+		return false;
+	}
 	int bufSize = ftell(fp);
```

Comment:
True - I'll update them. And again, a warning-free 64 bit build will help with this stuff - I'll have a look at doing that soon.